lib/cudadrv/occupancy.jl

```diff
@@ -59,7 +59,9 @@ function launch_configuration(fun::CuFunction; shmem::Union{Integer,Base.Callabl
         cuOccupancyMaxPotentialBlockSize(blocks_ref, threads_ref, fun, C_NULL, shmem, max_threads)
     elseif Sys.ARCH == :x86 || Sys.ARCH == :x86_64
         shmem_cint = threads -> Cint(shmem(threads))
-        cb = @cfunction($shmem_cint, Cint, (Cint,))
+        cb = lock(_shmem_cb) begin
```

Comment:
```suggestion
        cb = lock(_shmem_cb) do
```

I should not write code in my browser...